nemo/core/classes/exportable.py

```diff
@@ -153,9 +154,9 @@ def _export(
                         check_trace_input = [input_example]
                     else:
                         check_trace_input = check_trace
-
+                jitted_model = self
                 if format == ExportFormat.TORCHSCRIPT:
-
+                    # torch._C._jit_set_autocast_mode(False)
```

Comment:
This might be needed for models like conformer

Reply:
That was a leftover from debugging